templates/app/assets/stylesheets/spree/frontend/components/search/_search_bar.scss

```diff
@@ -1,5 +1,5 @@
 .search-bar {
-  border: solid 1px $color-border-medium;
+  border: 0;
```

Comment:
@mfrecchiami I copied the CSS from Solidus demo. Let me know if you find any issues with the CSS.